couchbase/tests/test_couchbase.py

```diff
@@ -0,0 +1,81 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+# stdlib
+from types import ListType
+
+import pytest
+
+from datadog_checks.couchbase import Couchbase
+
+from .common import (
+    HOST, PORT, CONFIG, AGENT_CONFIG,
+    CAMEL_CASE_TEST_PAIRS, EXTRACT_SECONDS_TEST_PAIRS, CAMEL_CASED_METRICS
+)
+
+couchbase = Couchbase('couchbase', CONFIG, AGENT_CONFIG)
+
+
+def test_camel_case_to_joined_lower(aggregator):
```

Comment:
Can you please remove the aggregator fixture argument from any function that doesn't actually use it?